internal/endtoend/testdata/nullable_subselect/mysql/query.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE foo (a int not null, b int);
```

Comment:
The line should be moved `schema.sql` and  change config to `"schema": "query.sql"`.